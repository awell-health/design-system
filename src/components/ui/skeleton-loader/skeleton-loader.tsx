import { cn } from '@/lib/utils';

function SkeletonLoader({ className, ...props }: React.HTMLAttributes<HTMLDivElement>) {
  return <div className={cn('skeleton w-full h-2', className)} {...props} />;
}

export { SkeletonLoader };
