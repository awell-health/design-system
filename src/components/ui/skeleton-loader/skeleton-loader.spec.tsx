import { describe, expect, it } from 'vitest';

import React from 'react';
import { SkeletonLoader } from './skeleton-loader';
import { render } from '@testing-library/react';

describe('SkeletonLoader', () => {
  it('match snapshot', () => {
    const { container } = render(<SkeletonLoader />);

    expect(container).toMatchSnapshot();
  });
});
