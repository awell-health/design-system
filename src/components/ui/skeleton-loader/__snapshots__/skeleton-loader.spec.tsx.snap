// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SkeletonLoader > match snapshot 1`] = `
<div>
  <div
    class="skeleton w-full h-2"
  />
</div>
`;
